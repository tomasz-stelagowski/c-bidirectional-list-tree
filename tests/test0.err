NODES: 2
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 7
NODES: 8
NODES: 8
NODES: 8
NODES: 7
NODES: 7
NODES: 7
NODES: 8
NODES: 8
NODES: 8
NODES: 6
NODES: 7
NODES: 7
NODES: 7
NODES: 8
NODES: 8
NODES: 8
NODES: 7
NODES: 6
NODES: 6
NODES: 6
NODES: 6
NODES: 7
NODES: 7
NODES: 7
NODES: 7
NODES: 7
