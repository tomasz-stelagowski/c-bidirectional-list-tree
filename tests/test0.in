ADD_NODE 0
ADD_NODE 1
ADD_NODE 1
ADD_NODE 1
ADD_NODE 1
ADD_NODE 4
RIGHTMOST_CHILD 2
ADD_NODE 5
RIGHTMOST_CHILD 4
RIGHTMOST_CHILD 1
DELETE_SUBTREE 3
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 5
ADD_NODE 1
RIGHTMOST_CHILD 7
RIGHTMOST_CHILD 1
DELETE_SUBTREE 5
ADD_NODE 6
RIGHTMOST_CHILD 4
RIGHTMOST_CHILD 1
ADD_NODE 0
RIGHTMOST_CHILD 10
RIGHTMOST_CHILD 0
DELETE_SUBTREE 8
DELETE_SUBTREE 10
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 6
RIGHTMOST_CHILD 9
ADD_NODE 4
RIGHTMOST_CHILD 4
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 2
RIGHTMOST_CHILD 6
