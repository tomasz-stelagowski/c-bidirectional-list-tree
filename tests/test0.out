OK
OK
OK
OK
OK
OK
-1
OK
6
5
OK
5
7
OK
-1
8
OK
OK
6
8
OK
-1
10
OK
OK
4
9
-1
OK
11
1
-1
9
